e2e/LongHaul/module/IotHub.cs

```diff
@@ -103,7 +103,7 @@ public async Task InitializeAsync(CancellationToken? ct = null)
         /// Frequently send telemetry messages to the hub.
         /// </summary>
         /// <param name="ct">The cancellation token</param>
-        public async Task SendTelemetryMessagesAsync(Logger logger, CancellationToken ct)
+        public async Task SendMessagesToRouteAsync(Logger logger, CancellationToken ct)
```

Comment:
Renamed since this method tests the messages-to-route API and not the send-telemetry API